src/app/accounting/search-journal-entry/search-journal-entry.component.html

```diff
@@ -38,6 +38,20 @@
     <input matInput [formControl]="transactionId">
   </mat-form-field>
 
+  <mat-form-field fxFlex="31%" (click)="submittedOnDateFromPicker.open()">
+    <mat-label>Submitted on Date From</mat-label>
+    <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="submittedOnDateFromPicker" [formControl]="submittedOnDateFrom">
+    <mat-datepicker-toggle matSuffix [for]="submittedOnDateFromPicker"></mat-datepicker-toggle>
+    <mat-datepicker #submittedOnDateFromPicker></mat-datepicker>
+  </mat-form-field>
+
+  <mat-form-field fxFlex="31%" (click)="submittedOnDateToPicker.open()">
+    <mat-label>Submitted on Date To</mat-label>
+    <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="submittedOnDateToPicker" [formControl]="submittedOnDateTo">
```

Comment:
We could use, as control, something like that
`[min]="formNameControl.value.submittedOnDateFrom"`